gradle.properties

```diff
@@ -43,7 +43,7 @@ stdlibMimeVersion=2.4.0
 stdlibTaskVersion=2.2.3-20220920-184600-c109457
 stdlibUrlVersion=2.2.2
 stdlibRandomVersion=1.3.0
-stdlibProtobufVersion=1.3.1-20221006-101100-9a65fe3
+stdlibProtobufVersion=1.3.2-20221006-142100-0685135
```

Comment:
Latest released version seems to be 1.3.1. Any reason to update this to the next developement timestamped version

Reply:
I was going to release grpc patch for 2201.2.x with protobuf:1.3.1 version here in https://github.com/ballerina-platform/module-ballerina-grpc/pull/1029

We can use the same version here as well right?

Reply:
Yes the version is 1.3.1. Here you have updated to `1.3.2`

Reply:
Updated